src/profiler/profiler.c

```diff
@@ -269,17 +272,15 @@ void xdebug_profiler_deinit()
 
 static inline void xdebug_profiler_function_push(function_stack_entry *fse)
 {
-	fse->profile.time += xdebug_get_utime();
-	fse->profile.time -= fse->profile.mark;
-	fse->profile.mark = 0;
-	fse->profile.memory += zend_memory_usage(0);
-	fse->profile.memory -= fse->profile.mem_mark;
+	fse->profile.nanotime += xdebug_get_nanotime() - fse->profile.nanotime_mark;
```

Comment:
Could I have `( … )` around the bit behind `+=` ? (and below in 277 too)

Reply:
I would never write them there, why do you want them there? If you assign a `a - b` to a variable, would you you parenthesis as well?

Reply:
fine, updated